src/ajax.js

```diff
@@ -543,7 +543,7 @@ jQuery.extend( {
 		if ( s.crossDomain == null ) {
 			urlAnchor = document.createElement( "a" );
 
-			// Support: IE <=8 - 11+, Edge 12 - 17 only
+			// Support: IE <=8 - 11+
```

Comment:
I removed mentions of Edge from support comments as well. Among other things, this is because many IE workarounds are now hidden behind an `isIE` check so a support comment mentioning Edge would be confusing as Edge wouldn't execute that code.

IE is quite special here as that's the only browser which we detect as a browser to work around its bugs instead of running support tests.